test/checks/color/color-contrast.js

```diff
@@ -157,4 +157,39 @@ describe('color-contrast', function () {
 		assert.equal(checkContext._data.missingData[0].reason, 'bgGradient');
 		assert.equal(checkContext._data.contrastRatio, 0);
 	});
+
+	it('should return undefined when there are elements overlapping', function () {
```

Comment:
The code was already in there and it was occurring for some nodes, this added a test for pre-existing code.